### ISSUE TITLE
"pending build requests › shows" e2e test fails on webkit sometimes

### ISSUE THREAD

**Issue by p12tic** [buildbot/buildbot#7410]
See e.g. here https://buildbot.buildbot.net/#/builders/127/builds/159:

```
[40/51] [webkit] › pendingbuildrequests.spec.ts:29:7 › pending build requests › shows
1) [webkit] › pendingbuildrequests.spec.ts:29:7 › pending build requests › shows ─────────────────
Test timeout of 30000ms exceeded.
Error: page.waitForURL: Navigation failed because page was closed!
=========================== logs ===========================
waiting for navigation until "load"
navigated to "http://localhost:8011/#/"
============================================================
at pages/builder.ts:29
27 | await BuilderPage.gotoBuildersList(page);
28 | await page.getByRole('link', {name: builder, exact: true}).click();
> 29 | await page.waitForURL(/\/#\/builders\/[0-9]+$/);
| ^
30 | await BasePage.waitUntilFinishedLoading(page);
31 | }
32 |
at Function.goto (/buildbot/buildbot-job/build/smokes-react/tests/pages/builder.ts:29:16)
at /buildbot/buildbot-job/build/smokes-react/tests/pendingbuildrequests.spec.ts:56:5
```

**Comment by mokibit** [comment]
@p12tic ,
fixed in  #7531.

**Comment by p12tic** [comment]
Thanks, test no longer fails in this way.
